src/analyses/threadEscape.ml

```diff
@@ -22,16 +22,10 @@ struct
   module V = VarinfoV
   module G = EscapeDomain.EscapedVars
 
-  let rec cut_offset x =
-    match x with
-    | `NoOffset    -> `NoOffset
-    | `Index (_,o) -> `NoOffset
-    | `Field (f,o) -> `Field (f, cut_offset o)
-
   let reachable (ask: Queries.ask) e: D.t =
     match ask.f (Queries.ReachableFrom e) with
     | a when not (Queries.LS.is_top a) ->
-      (* let to_extra (v,o) set = D.add (Addr.from_var_offset (v, cut_offset o)) set in *)
+      (* let to_extra (v,o) set = D.add (Addr.of_mval (v, cut_offset o)) set in *)
```

Comment:
If we leave this here, we probably also should leave `cut_offset`. Otherwise it's hard to understand what this used to do.

Reply:
These have been commented out since 2013, so I'll just remove them since not dropping index offsets doesn't seem to have been an issue in the last decade.